src/m365/spo/commands/folder/folder-add.spec.ts

```diff
@@ -154,13 +162,75 @@ describe(commands.FOLDER_ADD, () => {
     }));
   });
 
+  it('creates a folder with a specific color by color number', async () => {
+    const postStub = sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `${webUrl}/_api/foldercoloring/createfolder(DecodedUrl='${formatting.encodeQueryParameter(`${parentFolder}/${folderName}`)}', overwrite=false)`) {
+        return addResponse;
+      }
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { webUrl: webUrl, parentFolderUrl: parentFolder, name: folderName, color: colorNumber } });
+    assert.deepStrictEqual(postStub.lastCall.args[0].data, {
+      coloringInformation: {
+        ColorHex: `${colorNumber}`
+      }
+    });
+  });
+
+  it('creates a folder with a specific color by color name', async () => {
+    const postStub = sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `${webUrl}/_api/foldercoloring/createfolder(DecodedUrl='${formatting.encodeQueryParameter(`${parentFolder}/${folderName}`)}', overwrite=false)`) {
+        return addResponse;
+      }
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { webUrl: webUrl, parentFolderUrl: parentFolder, name: folderName, color: colorName } });
+    assert.deepStrictEqual(postStub.lastCall.args[0].data, {
+      coloringInformation: {
+        ColorHex: `${colorNumber}`
```

Comment:
We don't have to encapsulate it in string quotes anymore since the number is a string right now.
Same for the test above.